relayer/cli/src/commands/utils.rs

```diff
@@ -0,0 +1,11 @@
+use core::future::Future;
+
+/// block on future
+pub fn block_on<F: Future>(future: F) -> F::Output {
+    tokio::runtime::Builder::new()
+        .basic_scheduler()
+        .enable_all()
+        .build()
+        .unwrap()
+        .block_on(future)
+}
```

Comment:
You can probably get rid of this `block_on` method (might be relevant: https://github.com/informalsystems/tendermint-rs/pull/171, https://github.com/informalsystems/tendermint-rs/pull/173, https://github.com/informalsystems/tendermint-rs/pull/125#discussion_r388390015).

Reply:
Unfortunately, I have been unable to get `abscissa_tokio` to work, `abscissa_tokio::run` [just hangs](https://github.com/informalsystems/ibc-rs/blob/88e0c0cc49b8ea773b445b1bb95419a1f8117ab0/relayer/cli/src/commands/start.rs#L25-L26) (on my machine at least).

I haven't really put any time into debugging this, as I'd rather wait until we figure the new light client + relayer architecture, before spending time on that (`block_on` does the job just fine for now imho as it is called at the top level only).

Reply:
Makes sense!

Reply:
@romac just a note that we use this now for the query also.